### ISSUE TITLE
Numbered list/Bullet list drop down has accessibility issue (Place holder is not present)

### ISSUE THREAD

**Issue by Mahesh554** [ckeditor/ckeditor5#11796]
## 📝 Provide detailed reproduction steps (if any)

1. Open CKEditor which contains Numbered list/Bullet list tool items along with list properties drop down (You can use this link: https://ckeditor.com/docs/ckeditor5/latest/examples/builds-custom/full-featured-editor.html)
2. Hover over the Numbered list/Bullet list, place holder/help text is shown properly. Now hover over the dropdown beside the numbered list/bullet list dropdown

### ✔️ Expected result

When we hover over the numbered list/bullet list dropdown, some help text/place holder should be shown

### ❌ Actual result

No place holder/help text is shown

## 📃 Other details

* Browser: Any version of chrome/firefox
* OS: Windows 10
* First affected CKEditor version: CKEditor 5.x
* Installed CKEditor plugins: List properties, list
<img width="539" alt="HelperTextShownForBulletList" src="https://user-images.githubusercontent.com/28930402/169324010-aaa7840e-748d-43da-bfa0-f77080b5aea8.PNG">
Helper text is shown when I hover over the list icon
<img width="538" alt="NoHelperTextWhenHoverOverDropDown" src="https://user-images.githubusercontent.com/28930402/169324018-97f4d186-cb3c-4bfe-9f45-d3c7fc13f90b.PNG">
Helper text is not shown when I hover over the dropdown beside list icon

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by bbenjamin** [comment]
It may be more accurate to report this as an issue with splitbuttons. There's an additional issue I identified that _may_ be within this issue's scope, but I wouldn't be shocked if it warranted a separate issue.

When using a screenreader,  splitbutton disclosure buttons have no label, so AT conveys them as "menu pop up collapsed/open". There's nothing to indicate the visual group it is a part of.

**Comment by mlewand** [comment]
@bbenjamin @Mahesh554 to be sure I got it right, so it's about hovering/focusing the down arrow for split buttons, right? e.g.

No label when focusing down arrow in v33.0.0 ([permalink to demo](https://ckeditor.com/docs/ckeditor5/33.0.0/examples/builds/document-editor.html)):

![](https://user-images.githubusercontent.com/5353898/187196171-da8ffa3c-10c1-41ee-9ab5-b708d5173fc3.png)

We fixed it already in the last release (you can test it in [v35.0.1 demo](https://ckeditor.com/docs/ckeditor5/35.0.1/examples/builds/document-editor.html)):

![](https://user-images.githubusercontent.com/5353898/187196390-975bedfd-c806-40a1-b9fd-7fcea5db2b87.png)

That covers both visual tooltips and labels that would be conveyed by any sort of assistive technologies.

In case I'm missing something, please feel free to reopen the issue.

**Comment by Mahesh554** [comment]
Hi @mlewand, thanks for the update. I've checked in v35.0.1 and the issue is resolved.
